WBooks/Book_Details/BookDetailsView.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  BookDetailsView.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 8/30/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BookDetailsView: UIView {
+    
+    // MARK: - Properties
+    
+    @IBOutlet weak var logoImageView: UIImageView!
```

Comment:
What do you mean by logo? Would this be the book image? I would prefer another name, maybe `bookCoverImageView` or `bookImageView` or just `imageview`.